pkgs/applications/graphics/makeicns/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, lib
+, fetchurl
+, Foundation
+, AppKit
+, Cocoa
+}:
+
+stdenv.mkDerivation rec {
+  pname = "makeicns";
+  version = "1.4.10a";
+
+  src = fetchurl {
+    url = "https://distfiles.macports.org/makeicns/makeicns-${version}.tar.bz2";
+    sha256 = "1f6ydncnjfv86acaw4gcsa34ala65yjczhmrja6yscybhj6lpr0h";
+  };
+
+  dontConfigure = true;
+
+  buildInputs = [ Foundation AppKit Cocoa ];
+
+  patches = [
+    ./patch-IconFamily.m.diff
```

Comment:
This name is not very descriptive. It would help to know what is done instead of just knowing which file in the source code is being patched.

Reply:
This will now be removed since `makeicns` has been removed from the PR.